### PR TITLE
Updated interactive mode

### PR DESCRIPTION
**Pull request by Johnius**

### CODE REVIEW

File: servo [hunk 1]
```diff
@@ -343,10 +343,16 @@ def exc_format(e):
         return str(e) # print only the message but not the type
     return "{}: {}".format(type(e).__name__, str(e)) # print the exception type and message
 
+def pd(obj):
```
Comment (ekalosak) [inline servo:346]: Could have a more descriptive name &/or some inline comments re: functionality

File: servo [hunk 2]
```diff
@@ -355,6 +361,8 @@ def exec_command():
         return
 
     c = cmd['cmd']
+    print('Asked to {} with params {}.'.format(c, pd(cmd.get('param', None))), end=' ')
+    delay()
```
Comment (ekalosak) [inline servo:365]: Why is `delay()` called?

**Review by ekalosak — COMMENTED** [review]
Looks like aesthetic debugging output changes exclusively? Did I miss anything?

File: servo (hunk 1 above)
Comment (Johnius) [reply servo:346]: Following Lion's lean approach to naming things.

File: servo (hunk 2 above)
Comment (Johnius) [reply servo:365]: To let you decide when to react on a requested action.

**Comment by Johnius** [comment]
@pavel-georgiev Ok to merge?

**Pull request closed without merging**